app/controllers/api/todos_controller.rb

```diff
@@ -10,27 +10,26 @@ def index
     render "index.json.jb"
   end
 
-  def edit
-    @todo = Todo.find(params[:id])
+  def update
+    @todo = Todo.find_by(id: params[:id])
+    if @todo.update(todo_params)
+      @todo.save
+    end
   end
 
   def create
-    @todo = Todo.new(todo_params)
-
-    if @todo.save
-      render "index.json.jb"
-    else
-      render "error.json.jb"
+    todo = Todo.new(todo_params)
+    if todo.save
+      render json: { status: 200 }if @todo.save
```

Comment:
```suggestion
      render json: { status: 200 } if todo.save
```
スペースが必要そうですね！

@todoはtodoの間違いですね